.gitignore

```diff
@@ -0,0 +1,47 @@
+# VS Code #
+.vscode
+.vs/
+~$*
+*~
+
+# Sass #
+#.sass-cache/
+#*.css.map
+#*.sass.map
+#*.scss.map
+
+# Byte-compiled / Optimized #
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Flask #
+.flaskenv
+instance/
+.webassets-cache
+
+# pyenv #
+.python-version
+.Python
+
+# environments #
+ENV
+env
+env/
+.env
+env.bak/
+VENV
+venv
+venv/
+.venv
+venv.bak/
+
+# Installer Logs #
+pip-delete-this-directory.txt
+pip-log.txt
+
+# Unit Test #
+.cache
+.hypothesis/
+.pytest_cache/
+nosetests.xml

```

Comment:
While having all these here isn't wrong, I want you to focus on having the specific files you're ignoring, and remove other files not associated with your project, so the accessors know that you understand what you're doing.